Sources/FluentKit/FluentError.swift

```diff
@@ -1,13 +1,14 @@
 import Foundation
 
-public enum FluentError: Error, LocalizedError, CustomStringConvertible, CustomDebugStringConvertible {
+public enum FluentError: Error, LocalizedError, CustomNSError, CustomStringConvertible, CustomDebugStringConvertible {
```

Comment:
Why the `CustomNSError`? Is it for not getting those "Operation could not be completed" useless errors?

Reply:
Yes, it hopefully will improve error messages under some conditions on Linux; I can't prove conclusively that it does much of anything at all, but it isn't harmful or performance-affecting, so at worst it does nothing at all.

Reply:
I don't think we need `CustomNSError` then if those exact useless errors i mentioned is what you're worrying about:
https://github.com/apple/swift/issues/58724#issuecomment-1693488876

Reply:
So just to check, you want me to remove a layer of fallback redundancy on the basis that you went ahead and "well actually"'d a Swift bug filed by _@weissi_ of all people on the assumption he couldn't possibly know that already? Do I have that right? (Please ensure your mental rendering of my tone in this comment expresses sufficiently over-the-top exaggerated snark for maximum comedic effect 😂)

Reply:
I wouldn't dare to do such thing against @weissi, that was more for others who might read the issue 😅

But yes, I do want you to remove that layer of fallback redundancy since i think it's pretty useless and just makes the code across different repos inconsistent (do you really want to, even gradually, add that `CustomNSError` to all other types in other repos as well, just for the slightest chance that it might do something, although you can try the thing i mentioned in the issue and see that's all you need? 😛)